src/components/Card/Card.st.css

```diff
@@ -131,7 +132,7 @@
 
 .root > .infoContainer {
     box-sizing: border-box;
-    background-color: value(infoBGColor);
+    background-color: transparent;
```

Comment:
Why was this moved to the `root`?

Reply:
@jonathanadler for example if the user  wants to have transparent media background then we need to show the info background and not whats behind the Card so the easiest solution was to move the info background to root and make the info transparent

Reply:
So the info background color is the base background for the whole component?

Reply:
> So the info background color is the base background for the whole component?

yes, but after Johanna's comment i think that i'll add a third css override for the Card BG with default the same as Info